docs/components/checkbox-group-field/index.md

```diff
@@ -8,11 +8,73 @@ Provides a checkbox group to be used within forms. It yields [CheckboxFields](./
 
 ## Label
 
-Provide a string to `@label` to render the text into the `<legend>` of the fieldset.
+Required.
+
+Use either `@label` or `:label`.
```

Comment:
nit: To maybe be "modern Ember" friendly or more friendly to folks new to Ember, I wonder if being a bit more verbose here would be helpful:

```suggestion
Use either the `@label` component argument or the `:label` named block.
```